qiita_db/metadata_template.py

```diff
@@ -1561,9 +1561,16 @@ def update(self, md_template):
                                % ', '.join(samples_diff))
         columns_diff = set(new_map.columns) - set(current_map.columns)
         if columns_diff:
-            raise QiitaDBError('The new sample template differs from what is '
-                               'stored in database by these columns names: %s'
-                               % ', '.join(columns_diff))
+            queue = 'add_cols_%d' % self._id
```

Comment:
This is consistent with the existing codebase, and a refactor is out of scope. Issue #1020 created for the refactor.